v0/destinations/facebook_pixel/transform.js

```diff
@@ -527,6 +527,23 @@ const responseBuilderSimple = (message, category, destination) => {
     }
   }
 
+  // content_category should only be a string ref: https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data
+
+  if (
+    customData &&
+    customData.content_category &&
+    typeof customData.content_category !== "string"
+  ) {
+    if(Array.isArray(customData.content_category)) {
+      customData.content_category = customData.content_category.map(String).join('-')
```

Comment:
```suggestion
      customData.content_category = customData.content_category.map(String).join(',')
```

Reply:
pushing it in a seperate commit, as need to update the test case

Reply:
resolving conversation as changes are approved.